contracts/FiatToken.cdc

```diff
@@ -1070,218 +932,217 @@ pub contract FiatToken: FungibleToken {
-
-        // Setup the Pauser
-        pubKeyAttrs = []
-        i = 0
-        while i < pauserPubKeys.length {
-            let pka = OnChainMultiSig.PubKeyAttr(sa: pauserPubKeysAlgos[i], w: pauserPubKeysWeights[i])
-            pubKeyAttrs.append(pka)
-            let key = PublicKey(
-                publicKey: pauserPubKeys[i].decodeHex(), 
-                signatureAlgorithm: SignatureAlgorithm(rawValue: pauserPubKeysAlgos[i]) ?? panic ("Invalid signature algo")
-            )
-            pauserAccount.keys.add(
-                publicKey: key,
-                hashAlgorithm: HashAlgorithm.SHA3_256,
-                weight: pauserPubKeysWeights[i]
-            )
-            i = i + 1
-        }
-        pauserAccount.save(<- self.createNewPauser(publicKeys: pauserPubKeys, pubKeyAttrs: pubKeyAttrs), to: self.PauserStoragePath)
-        pauserAccount.link<&Pauser{OnChainMultiSig.PublicSigner}>(self.PauserPubSigner, target: self.PauserStoragePath)
-        pauserAccount.link<&Pauser{ResourceId}>(self.PauserUUIDPubPath, target: self.PauserStoragePath)
-        pauserAccount.link<&Pauser{PauseCapReceiver}>(self.PauserCapReceiverPubPath, target: self.PauserStoragePath)
-
-        // Assign the admin capabilities
-        let adminExecutorRef = contractAccount.borrow<&FiatToken.AdminExecutor>(from: self.AdminExecutorStoragePath)
-            ?? panic("cannot borrow AdminExecutor from storage")
-        let ownerExecutorRef = contractAccount.borrow<&FiatToken.OwnerExecutor>(from: self.OwnerExecutorStoragePath)
-            ?? panic("cannot borrow OwnerExecutor from storage")
-        adminExecutorRef.changeAdmin(to: adminAccount.address, newPath: initialAdminCapabilityPrivPath)
-        ownerExecutorRef.reassignOwner(to: ownerAccount.address, newPath: initialOwnerCapabilityPrivPath)
-        ownerExecutorRef.reassignMasterMinter(to: masterMinterAccount.address, newPath: initialMasterMinterCapabilityPrivPath)
-        ownerExecutorRef.reassignBlocklister(to: blocklisterAccount.address, newPath: initialBlocklisterCapabilityPrivPath)
-        ownerExecutorRef.reassignPauser(to: pauserAccount.address, newPath: initialPauserCapabilityPrivPath) 
-
-        // Create a Vault with the initial totalSupply
-        let vault <- create Vault(balance: self.totalSupply)
-        self.account.save(<-vault, to: self.VaultStoragePath)
-
-        // Create public capabilities to the vault
-        contractAccount.link<&FiatToken.Vault{FungibleToken.Receiver}>(self.VaultReceiverPubPath, target: self.VaultStoragePath)
-        contractAccount.link<&FiatToken.Vault{FungibleToken.Balance}>(self.VaultBalancePubPath, target: self.VaultStoragePath)
-        contractAccount.link<&FiatToken.Vault{FiatToken.ResourceId}>(self.VaultUUIDPubPath, target: self.VaultStoragePath)
-
-        // Emit the TokensInitialized event
-        emit TokensInitialized(initialSupply: self.totalSupply)
-
-    }
-
-}
+    ) {	
+
```

Comment:
I was updating all the init logic to make deploying and testing a bit easier. It doesn't get run during upgrades anyway, so none of this will affect anything on testnet and mainnet when the contract is upgraded